manifests/server/conf.pp

```diff
@@ -350,7 +350,7 @@
   }
 
   $subsystem_array = split($subsystem, ' +')
```

Comment:
Why split at all if all you are doing is joining it back together?

Reply:
I don't remember, but we'll go with the "if it ain't broke" approach right now.